__mocks__/@prisma/client.js

```
@@ -0,0 +1,285 @@
+   * transactional client (same API here). If passed an array, run queries
+   * sequentially and return results.
+   */
+  async $transaction(arg) {
+    if (typeof arg === 'function') {
+      // Call callback with this proxy so model getters work inside
+      return await arg(this);
+    }
+    if (Array.isArray(arg)) {
+      const results = [];
+      for (const op of arg) {
+        if (typeof op === 'function') results.push(await op(this));
```

ESLint

[error] Replace `'function'` with `"function"`

(prettier/prettier)